tools/wiki_entities/README.md

```diff
@@ -12,8 +12,27 @@ pip install mafan
 pip install textblob
 ```
 
-4. Run the following two scripts. These two scripts contain some simple rules designed for English, Spanish, and Chinese. You may want to modify them to fit the new language **X**. You will find "**X**/wiki_quality.txt" and "**X**/wiki_all.txt".
+4. Run the script. This script contain some simple rules designed for English, Spanish, and Chinese. You may want to modify them to fit the new language **X**. You will find "**X**/wiki_quality.txt" and "**X**/wiki_all.txt".
 ```
-python wiki_all_phrase_filter.py -lang X
-python high_quality_phrase_filter.py -lang X
+python extract_entities.py --lang X --quality
+python extract_entities.py --lang X --all
+```
+
+4.1. If neither `--quality` nor `--all` intend to be used, the script also supports custom arguments:
+```
+python extract_entities.py -h
+
+usage: extract_entities.py [-h] [--lang LANG] [--min_sup MIN_SUP] [--min_perc MIN_PERC] [--stopwords_orig] [--stopwords_end] [--only_alpha] [--quality] [--all]
+
+optional arguments:
+  -h, --help           show this help message and exit
+  --lang LANG          language/dir containing files
+  --min_sup MIN_SUP    minimum support for considering entity
+  --min_perc MIN_PERC  minimum percentage for considering entity
+  --stopwords_orig     whether tokens starting w/ stopwords or w/ high proportion of stopwords should be ignored
```

Comment:
need a separate parameter for `w/ high proportion of stopwords should be ignored`

Reply:
Agree. Will do.